modules/ossm-creating-istio-cni-project-using-console.adoc

```diff
@@ -0,0 +1,17 @@
+:_mod-docs-content-type: Procedure
+[id="ossm-creating-istio-cni-project"]
+= Creating the istio-cni project using the web console
+:context: ossm-creating-istio-cni-project-using-console
+
+.Prerequisties
+
+
+.Procedure
+
+. In the {ocp-product-title} web console, click *Home* -> *Projects*.
+
+. Click *Create Project*.
+
+. At the prompt, you must enter a name for the project in the Name field. For example, `istio-cni`. The Operator deploys {istio} to the project you specify. The other fields provide supplementary information and are optional.
```

Comment:
Here says: `The Operator deploys {istio} to the project you specify`, but the step is to create the `istio-cni`. But, please take a look at the comments that I added to the istio-system project creation. I think this comments should be moved

Reply:
I fixed things so the attribute resolves now.

I'll run the other comment by the team just to get more opinions.

Reply:
I think we should remove the sentence `The Operator deploys ...`

Reply:
@fjglira I just re-read this and now I realize what you mean. Yes, I agree. Change was made.